openslides/motions/config_variables.py

```diff
@@ -75,7 +60,7 @@ def get_config_variables():
 
     yield ConfigVariable(
         name="motions_default_line_numbering",
-        default_value="none",
+        default_value="outside",
```

Comment:
I don't like it, but feel free to change it.

Reply:
As discussed, we will change the default to 'outside'.